sql-statements/sql-statement-admin-show-ddl.md

```diff
@@ -44,7 +44,32 @@ mysql> ADMIN SHOW DDL;
 
 ### `ADMIN SHOW DDL JOBS`
 
-To view all the results in the current DDL job queue (including tasks that are running and waiting to be run) and the last ten results in the completed DDL job queue, use `ADMIN SHOW DDL JOBS`:
+The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the current DDL job queue, including running and queuing tasks, as well as the latest ten results in the completed DDL job queue. The returned result fields are described as follows:
+
+- `JOB_ID`: Each DDL operation corresponds to a DDL job. `JOB_ID` is globally unique.
+- `DB_NAME`: The name of the database where the DDL operation is performed.
+- `TABLE_NAME`: The name of the table where the DDL operation is performed.
+- `JOB_TYPE`: The type of DDL operation.
+- `SCHEMA_STATE`: The current state of the schema object that the DDL operates on. If `JOB_TYPE` is `ADD INDEX`, it is the state of the index; if `JOB_TYPE` is `ADD COLUMN`, it is the state of the column; if `JOB_TYPE` is `CREATE TABLE`, it is the state of the table. Common states include the following:
+
+    - `none`: indicates that it does not exist. Generally, after the `DROP` operation or after the `CREATE` operation fails and rolls back, it will become the `none` state.
+    - `delete only`, `write only`, `delete reorganization`, `write reorganization`: These four states are intermediate states. For their specific meanings, refer to [How the Online DDL Asynchronous Change Works in TiDB](/ddl-introduction.md#how-the-online-ddl-asynchronous-change-works-in-tidb). As the intermediate state conversion is fast, these states are generally not visible during operation. Only when performing `ADD INDEX` operation can the `write reorganization` state be seen, indicating that index data is being added.
+    - `public`: indicates that it exists and is available to users. Generally, after `CREATE TABLE` and `ADD INDEX` (or `ADD COLUMN`) operations are completed, it will become the `public` state, indicating that the newly created table, column, and index can be read and written normally.
+
+- `SCHEMA_ID`: The ID of the database where the DDL operation is performed.
+- `TABLE_ID`: The ID of the table where the DDL operation is performed.
+- `ROW_COUNT`: When performing the `ADD INDEX` operation, it is the number of data rows that have been added.
+- `START_TIME`: The start time of the DDL operation.
+- `STATE`: The state of the DDL operation. Common states include the following:
+
+    - `queueing`: indicates that the operation job has entered the DDL job queue but has not been executed because it is still waiting for the previous DDL job to complete. Another reason might be that after executing the `DROP` operation, it will become the `none` state, but it will soon be updated to the `synced` state, indicating that all TiDB instances have been synchronized to that state.
```

Comment:
```suggestion
    - `queueing`: indicates that the operation job has entered the DDL job queue but has not been executed because it is still waiting for an earlier DDL job to complete. Another reason might be that after executing the `DROP` operation, it will become the `none` state, but it will soon be updated to the `synced` state, indicating that all TiDB instances have been synchronized to that state.
```